include/fcntl.h

```diff
@@ -15,10 +15,8 @@
 #define O_DIRECT     040000
 #define O_NOATIME  01000000
 #elif ((defined CONFIG_ARCH_ARM_64) || (defined CONFIG_ARCH_ARM_32))
-#undef O_NONBLOCK
-#define O_NONBLOCK    04000
 #define O_NOFOLLOW  0100000
-#define O_DIRECTORY  040000
```

Comment:
Please note that `O_DIRECTORY` is a flag and should be only a single bit set. The value `0123456` is setting too many bits in its binary representation.